common/src/main/java/com/wynntils/models/map/pois/SeaskipperPoi.java

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.map.pois;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.models.map.PoiLocation;
+import com.wynntils.models.map.type.DisplayPriority;
+import com.wynntils.utils.colors.CommonColors;
+import com.wynntils.utils.colors.CustomColor;
+import com.wynntils.utils.render.buffered.BufferedFontRenderer;
+import com.wynntils.utils.render.buffered.BufferedRenderUtils;
+import com.wynntils.utils.render.type.HorizontalAlignment;
+import com.wynntils.utils.render.type.TextShadow;
+import com.wynntils.utils.render.type.VerticalAlignment;
+import net.minecraft.client.renderer.MultiBufferSource;
+import net.minecraft.world.phys.Vec3;
+
+public class SeaskipperPoi implements Poi {
+    private final String destination;
+    private final PoiLocation destinationCenter;
+    private boolean isPlayerAtPoi;
+    private final int combatLevel;
+    private final int width;
+    private final int height;
+    private final int startX;
+    private final int startZ;
+    private final int endX;
+    private final int endZ;
+    private float renderedX;
+    private float renderedX2;
+    private float renderedY;
+    private float renderedY2;
+
+    public SeaskipperPoi(String destination, int combatLevel, int startX, int startZ, int endX, int endZ) {
+        this.destination = destination;
+        this.combatLevel = combatLevel;
+        this.startX = startX;
+        this.startZ = startZ;
+        this.endX = endX;
+        this.endZ = endZ;
+        this.width = endX - startX;
+        this.height = endZ - startZ;
+        this.destinationCenter = new PoiLocation(startX + width / 2, null, startZ + height / 2);
+    }
+
+    @Override
+    public PoiLocation getLocation() {
+        return destinationCenter;
+    }
+
+    @Override
+    public DisplayPriority getDisplayPriority() {
+        return DisplayPriority.HIGHEST;
+    }
+
+    @Override
+    public boolean hasStaticLocation() {
+        return false;
```

Comment:
Shouldn't this be true? (Not that it matters, if the seaskipper map does not implement waypoint selection)